azure/activity_logs_monitoring/index.js

```diff
@@ -102,6 +106,10 @@ function handleJSONArrayLogs(sender, context, logs, logsType) {
                 return;
             }
         }
+        // If the message is a buffer object, the data type has been set to binary.
+        if (logsType == BUFFER_ARRAY) {
+            message = JSON.parse(message.toString());
```

Comment:
Let's call this JSON.parse within a try/catch similar to what we do for JSON_STRING_ARRAY in case the string passed in isn't actually json. I'm not sure that we can assume it'll always be JSON